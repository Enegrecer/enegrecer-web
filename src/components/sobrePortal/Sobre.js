import React from 'react';
import Colaborador from './componentes/Colaborador';
import { colaboradores } from './colaboradores';
import './css/sobre.css';

const Sobre = () => (
  <div className="row sobre">
    <div className="secao_sobre">
      <div className="col s12 m6 foto_sobre">
        <img alt="Foto ilustrativa do Verdade Seja Dita" src="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fsobre.jpg?alt=media&token=3646a7f1-71af-4443-b1fb-36ec5dfa3b89" />
      </div>
      <div className="col s12 m6 conteudo">
        <div className="detalhe_titulo">
          <h1 id="sobre">Sobre</h1>
          <div className="traco-vermelho" />
        </div>
        <h1>Verdade Seja</h1>
        <h1>Dita</h1>
        <p>
      Somos mulheres e homens e definimos a nossa atuação na defesa e promoção
      dos direitos da população negra e na construção de uma sociedade onde os
      valores de justiça, equidade e solidariedade são fundamentais.
      Consideramos o racismo e a discriminação racial como mecanismos
      combinados que estruturam as relações sociais, cujos resultados diretos
      se expressam em prejuízos para as negras e negros. O racista brasileiro
      não tem nada de cordial e amistoso como muitos imaginam e sustentam. Pelo
      contrário. Nosso objetivo é criar e aplicar novas tecnologias para a
      luta política dos negros e negras contra o racismo; produzindo sobre o
      contexto atual das questões envolvendo os nossos direitos.
        </p>
      </div>
    </div>

    <div className="descricao_voluntario">
      <div className="col s12 secao_voluntarios">
        <div className="conteudo">
          <div className="col s12 m4 titulo_voluntarios">
            <div className="detalhe_titulo">
              <h1 id="quem-passou-por-aqui">Quem</h1>
              <div className="traco-vermelho" />
            </div>
            <h1>passou por</h1>
            <h1>aqui</h1>
          </div>

          <p className="col s12 m8 texto_voluntarios">
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
        minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
        ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
        sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
        mollit anim id est laborum.
          </p>
        </div>
      </div>

      <div className="col s12 colaboradores">
        {
          colaboradores().map(colaborador => (
            <Colaborador
              key={colaborador.nome}
              nome={colaborador.nome}
              papel={colaborador.papel}
              foto={colaborador.foto}
            />
          ))
        }
      </div>
    </div>
  </div>
);

export default Sobre;
