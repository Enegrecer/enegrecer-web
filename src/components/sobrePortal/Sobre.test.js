import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Sobre from './Sobre';

describe('Sobre', () => {
  const colaboradoresStub = {
    colaborador: {
      nome: 'Will Smith',
      papel: 'Desenvolvedor',
      foto: 'http://google.com',
    },
  };

  const wrapper = shallow(<Sobre colaboradores={colaboradoresStub} />);

  it('possui a mesma estrutura', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
