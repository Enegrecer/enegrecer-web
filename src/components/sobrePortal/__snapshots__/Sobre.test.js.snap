// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sobre possui a mesma estrutura 1`] = `
<div
  className="row sobre"
>
  <div
    className="secao_sobre"
  >
    <div
      className="col s12 m6 foto_sobre"
    >
      <img
        alt="Foto ilustrativa do Verdade Seja Dita"
        src="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fsobre.jpg?alt=media&token=3646a7f1-71af-4443-b1fb-36ec5dfa3b89"
      />
    </div>
    <div
      className="col s12 m6 conteudo"
    >
      <div
        className="detalhe_titulo"
      >
        <h1
          id="sobre"
        >
          Sobre
        </h1>
        <div
          className="traco-vermelho"
        />
      </div>
      <h1>
        Verdade Seja
      </h1>
      <h1>
        Dita
      </h1>
      <p>
        Somos mulheres e homens e definimos a nossa atuação na defesa e promoção dos direitos da população negra e na construção de uma sociedade onde os valores de justiça, equidade e solidariedade são fundamentais. Consideramos o racismo e a discriminação racial como mecanismos combinados que estruturam as relações sociais, cujos resultados diretos se expressam em prejuízos para as negras e negros. O racista brasileiro não tem nada de cordial e amistoso como muitos imaginam e sustentam. Pelo contrário. Nosso objetivo é criar e aplicar novas tecnologias para a luta política dos negros e negras contra o racismo; produzindo sobre o contexto atual das questões envolvendo os nossos direitos.
      </p>
    </div>
  </div>
  <div
    className="descricao_voluntario"
  >
    <div
      className="col s12 secao_voluntarios"
    >
      <div
        className="conteudo"
      >
        <div
          className="col s12 m4 titulo_voluntarios"
        >
          <div
            className="detalhe_titulo"
          >
            <h1
              id="quem-passou-por-aqui"
            >
              Quem
            </h1>
            <div
              className="traco-vermelho"
            />
          </div>
          <h1>
            passou por
          </h1>
          <h1>
            aqui
          </h1>
        </div>
        <p
          className="col s12 m8 texto_voluntarios"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
      </div>
    </div>
    <div
      className="col s12 colaboradores"
    >
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fadriana-hoher.jpg?alt=media&token=37628e1b-369c-42ac-9abf-9e73a702bb53"
        key="Adriana Höher"
        nome="Adriana Höher"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Falisson-pereira.jpg?alt=media&token=f317aa99-e343-474b-838f-c3ecae379e17"
        key="Alisson Pereira"
        nome="Alisson Pereira"
        papel="QA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fayrton-araujo.jpg?alt=media&token=96c23804-c0f5-4efd-b6cc-906b65fed757"
        key="Ayrton Araújo"
        nome="Ayrton Araújo"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fbarbara-sanches.jpg?alt=media&token=4ef955c0-55ea-4bf1-99d6-0ad89d5907fd"
        key="Bárbara Sanches"
        nome="Bárbara Sanches"
        papel="BA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fbarbara-tomaz.jpg?alt=media&token=c350dba2-eb28-46ce-a867-48d398a02d17"
        key="Bárbara Tomaz"
        nome="Bárbara Tomaz"
        papel="XD"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fbruna-castelo.jpg?alt=media&token=8d165c0d-41ed-4048-93c9-5bc5d0c95898"
        key="Bruna Castelo"
        nome="Bruna Castelo"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fcaique-peixoto.jpeg?alt=media&token=787a64be-811b-473b-a019-6dc95a2d5006"
        key="Caique Peixoto"
        nome="Caique Peixoto"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fcarine-monteiro.jpg?alt=media&token=a5485e22-02a9-4ba9-8619-672ad8d89a10"
        key="Carine Monteiro"
        nome="Carine Monteiro"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fdaniel-faria.png?alt=media&token=a8eb3e73-ac03-41d4-86d4-4533528a27eb"
        key="Daniel Faria"
        nome="Daniel Faria"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fdebora-beda.jpg?alt=media&token=d5fc5cea-0f07-4cf6-8d8f-8ac00c258e46"
        key="Débora Beda"
        nome="Débora Beda"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fedlaine-zamora.jpeg?alt=media&token=49355897-2c37-4859-a82a-fc2220d29040"
        key="Edlaine Zamora"
        nome="Edlaine Zamora"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fegileu-trajano.jpg?alt=media&token=8b5dabee-9ba9-4eb3-a158-7a65c60db0cb"
        key="Egileu Trajano"
        nome="Egileu Trajano"
        papel="QA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Ferika-silva.jpg?alt=media&token=adf0ebc8-cf6a-4967-b42c-a17a4383d567"
        key="Erika Silva"
        nome="Erika Silva"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Ffernando_machado.jpg?alt=media&token=def29f87-7902-4738-a90d-5d77f764f45a"
        key="Fernando Machado"
        nome="Fernando Machado"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fgabriel-pereira.jpg?alt=media&token=d41141ab-40d8-4789-b193-102825ce55ad"
        key="Gabriel Pereira"
        nome="Gabriel Pereira"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fgabriela-mattos.png?alt=media&token=615c00dc-1e79-40e5-856c-6262178fa03e"
        key="Gabriela Mattos"
        nome="Gabriela Mattos"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fhelena-silva.jpg?alt=media&token=c9f108da-f222-4cf9-bb78-2ca0889e91dd"
        key="Helena Silva"
        nome="Helena Silva"
        papel="PM"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fhenrique-brighenti.jpg?alt=media&token=5a38dc50-7e6e-436c-bc41-7ab28ff64853"
        key="Henrique Brighenti"
        nome="Henrique Brighenti"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Finessa-luerce.jpg?alt=media&token=b5d2acc7-f856-4056-ac74-6f4cfd25ef01"
        key="Inessa Luerce"
        nome="Inessa Luerce"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fizael-effemberg.jpg?alt=media&token=f6478b20-9702-485b-9282-446eafe5662a"
        key="Izael Effemberg"
        nome="Izael Effemberg"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fjoao-moreira.jpeg?alt=media&token=81fab300-2e6c-4155-a994-3d002288406b"
        key="João Moreira"
        nome="João Moreira"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fkamila-serafim.jpg?alt=media&token=23af266e-f6fe-4979-8a97-a5a697cd9135"
        key="Kamila Serafim"
        nome="Kamila Serafim"
        papel="QA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fkarla-falcao.jpeg?alt=media&token=d84ff7ad-e2ce-43c1-ad24-1a9df2c6b142"
        key="Karla Falcão"
        nome="Karla Falcão"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Flaura-bella.jpg?alt=media&token=842709e1-04e5-46ce-b1fd-491cacfdbafd"
        key="Laura de Bella"
        nome="Laura de Bella"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fleandro-vicente.jpeg?alt=media&token=ec9f54e6-c59d-4c81-8c23-8e93f24a82e7"
        key="Leandro Vicente"
        nome="Leandro Vicente"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Florena-souza.png?alt=media&token=68743873-9ea1-4ae8-b5c8-7baf94a2c214"
        key="Lorena Souza"
        nome="Lorena Souza"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fluciana-montenegro.jpg?alt=media&token=b9a49a6b-de78-4837-a480-d39cf3115f6c"
        key="Luciana Montenegro"
        nome="Luciana Montenegro"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fmarcella-souza.jpg?alt=media&token=7b38e3b3-e993-461d-a0c9-93f4b7a96b99"
        key="Marcella Souza"
        nome="Marcella Souza"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fmariane-ferroni.jpg?alt=media&token=a0c497c5-eaf8-452a-a20c-dc43df4ce246"
        key="Mariane Ferroni"
        nome="Mariane Ferroni"
        papel="BA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fmila-orrico.jpg?alt=media&token=f09a3af0-30a7-45df-97a4-1d3c6e652b79"
        key="Mila Orrico"
        nome="Mila Orrico"
        papel="BA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fnathalia-oliveira.png?alt=media&token=0199df2c-5e21-448b-a562-5468c896aaad"
        key="Nathalia Oliveira"
        nome="Nathalia Oliveira"
        papel="PM"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fnayara-souza.png?alt=media&token=7f38d6a0-8903-45bf-9846-929795b60102"
        key="Nayara Souza"
        nome="Nayara Souza"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fpaulo-gama.jpg?alt=media&token=6b9e8cba-53cc-4c50-84b9-cad489fc37be"
        key="Paulo Gama"
        nome="Paulo Gama"
        papel="DEV"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fpriscila-rocha.jpg?alt=media&token=04d3c50b-73f2-4bb1-b108-84286ab104d3"
        key="Priscila Rocha"
        nome="Priscila Rocha"
        papel="PM"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Frafaella-souza.jpeg?alt=media&token=68d28c4d-ef4b-4306-9382-46e8d61a262e"
        key="Rafaella Souza"
        nome="Rafaella Souza"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Frosangela-mendes.jpg?alt=media&token=42c77d11-e328-4f62-90d5-a8ca6f908974"
        key="Rosangela Mendes"
        nome="Rosangela Mendes"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Froselma-mendes.png?alt=media&token=18d57eac-0d6d-4e04-a5d2-0fa3af76232e"
        key="Roselma Mendes"
        nome="Roselma Mendes"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fsergio-luis.jpg?alt=media&token=5f03b54b-5b15-4cbb-8bda-b20cf9750e33"
        key="Sérgio Luis"
        nome="Sérgio Luis"
        papel="BA"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Ftatiane-nogueira.png?alt=media&token=4ec16519-9158-41d9-b279-ca0b6d98e8ea"
        key="Tatiane Aguirres Nogueira"
        nome="Tatiane Aguirres Nogueira"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Fthais-lopes.jpg?alt=media&token=6733cfb8-d256-4dfb-9673-d974683cac4f"
        key="Thais Lopes"
        nome="Thais Lopes"
        papel="DEVa"
      />
      <Colaborador
        foto="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fpessoas%2Ftiago-megale.jpg?alt=media&token=b50b0142-d9d4-42ce-ba81-d2aaa6635cda"
        key="Tiago Megale"
        nome="Tiago Megale"
        papel="BA"
      />
    </div>
  </div>
</div>
`;
